packages/fx-core/src/plugins/resource/bot/plugin.ts

```diff
@@ -135,18 +135,9 @@ export class TeamsBotImpl implements PluginImpl {
     );
     await handler?.start(ProgressBarConstants.PROVISION_STEP_START);
 
-    // 0. Check Resource Provider
-    const azureCredential = await this.getAzureAccountCredenial();
-    const rpClient = factory.createResourceProviderClient(
-      azureCredential,
-      this.config.provision.subscriptionId!
-    );
-    await factory.ensureResourceProvider(rpClient, AzureConstants.requiredResourceProviders);
-
     // 1. Do bot registration.
     await handler?.next(ProgressBarConstants.PROVISION_STEP_BOT_REG);
-    const botAuthCreds = await this.createOrGetBotAppRegistration();
-
+    await this.createOrGetBotAppRegistration();
```

Comment:
Just a reminder, if there is no other place require GetBotAppRegistration() logic, may be we can refactor it as just createXXX()

Reply:
done